app/screens/settings-screen/settings-screen.tsx

```diff
@@ -156,6 +158,12 @@ export const SettingsScreenJSX: ScreenType = (params: SettingsScreenProps) => {
     securityAction,
     resetDataStore,
   } = params
+  const copyToClipBoard = (username) => {
+    Clipboard.setString(LN_PAGE_DOMAIN + username)
+    Clipboard.getString().then((data) =>
+      toastShow(translate("tippingLink.copied", { data })),
```

Comment:
Could you add an optional parameter to [toastShow ](https://github.com/GaloyMoney/galoy-mobile/blob/5d0ac53558c3655fddfea9642e17fd9c6d813f23/app/utils/toast.ts#L4) for the backgroundColor, so that the toast can be a more friendly color than red. Maybe green?

Reply:
I changed it to the lightBlue options because I personally liked it better/matches the app's branding a bit better but happy to make it green if preferable.